RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+
+![Supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceipts.png "[Supplier receipts]")
+
+1.  Click the *Supplier receipt* menu entry.  
+The *Supplier receipts* window is displayed. The external receipts uploaded in the *New supplier receipts* folder in the *Documents* module are displayed in the *Acquire* tab.
+
+    ![Acquire supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsAcquire.png "[Acquire supplier receipts]")
+
+    > [Info] If there are no receipts to be booked, the message *No documents left to be registered* is displayed.
+
+  [comment]: <> (They do not appear in Acquire if located in New supplier receipts folder, but in the New documents. Check why/if correct/if it is a bug?)
+
+2. Enter the total invoice value in the *Turnover* field.
+
+  > [Info] Currency exchange rate can also be adjusted if needed in the *Exchange rate* field.
+
+3. Enter the corresponding booking details in the *ContraAcct* and *Supplier* fields.
+
+4. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY or DD.MM.YY.
+
+5. Enter a description account number in the *Posting text* field.
+
+  > [Info] Note that the fields *ContraAcct*, *Date*,  *Supplier* and *Posting text* must be filled out to be able to acquire a receipt.
+
+6. Enter any other relevant booking information in the corresponding fields.
+
+  > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Reference1* field, as the system identifies and groups together any subsequent bookings taking this field as reference.
+
+7. Select the appropriate payment target for the receipt in the *Payment target* drop-down list.
+
+  ![Acquire supplier receipts](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsAcquire02.png "[Acquire supplier receipts]")
+
+8. A split booking can also be performed with the [SPLIT] button, if necessary. For detailed information, see [Split a booking](./09_SplitBooking.md).
+
+9. Click the [ACQUIRE] button.  
+The supplier receipt is acquired and displayed in the *Overview* tab.  
+
+  ![Overview](../../Assets/Screenshots/RetailSuiteAccounting/Book/SupplierReceiptsOverview.png "[Overview]")
+
+10. The acquired receipts can be searched for using the *QUICK SEARCH* or the *ACTIVE FILTER* context menus.
+
+  > [Info] To be able to enter the supplier receipt in the system, they need to be released in the *Purchasing* module first. For detailed information, see [Check supplier receipts](#to-be-completed). Once they have been released in the *Purchasing* module, the status changes to **released** (colour yellow-green) in the *Status* column.
+
+11. Click the *Financial accounting takeover* tab and select the receipt to be entered in the system.
+
+  > [Info] When selecting a receipt, the receipt data are displayed in the entry line. Some changes can still be made here before takeover and saved with the [SAVE] button.
```

Comment:
Grundsätzliche Terminologie: entry line vs. input line -> entscheide dich für eins und kontrolliere das nochmal in allen Dokumenten.